docs/images/add_rule_ha.jsseq

```diff
@@ -0,0 +1,8 @@
+portal->perseo: visual rule
```

Comment:
Whait is .jsseq? (Just curiosity)

Reply:
(By the way it seems add_rule_ha.jsseq is not refered by any other file. Is it actually used?)

Reply:
That must be Sketch files, the editing program that was used for the new images. I've already deleted the files. 

Fixed in 45e10846a172668d524e38b689a570f702c07316

Reply:
Is ok to keep them as "source" of the current diagrams. I was just asking :)

Reply:
Did you finally re-added the files at the end? It's important to have the sources there in case somebody needs to modify in the future.